src/main/menus/app.ts

```diff
@@ -176,6 +177,58 @@ export function createTemplate(config: Config, updateManager: UpdateManager) {
         });
     }
 
+    if (DeveloperMode.enabled()) {
+        devToolsSubMenu.push(separatorItem);
```

Comment:
Each call to `push()` is for a unique item, with different actions for each one. Currently it's all just `toggle()` but we could have others.

Let's say we never do that: We'd have to create an array of values that include the translation string, the default translation and the id anyways. To me this decreases readibility, because then you have to backtrack through past the call to `push` to the array and figure out what that's going.

This seems pretty clear to me to see a menu item added for each menu item, and is also how the rest of the menu items work (outside of servers which aren't hardcoded).

One thing that could be made nicer is calling `push()` on an array of `MenuItem[]` instead of `push()` each time, so I'll change that.